gitlab/v4/objects/projects.py

```diff
@@ -1055,3 +1063,17 @@ class ProjectStorageManager(GetWithoutIdMixin, RESTManager):
 
     def get(self, **kwargs: Any) -> ProjectStorage:
         return cast(ProjectStorage, super().get(**kwargs))
+
+
+class ProjectCiLint(RESTObject):
+    pass
+
+
+class ProjectCiLintManager(GetWithoutIdMixin, CreateMixin, RESTManager):
+    _path = "/projects/{project_id}/ci/lint"
+    _obj_cls = ProjectCiLint
+    _from_parent_attrs = {"project_id": "id"}
+    # https://docs.gitlab.com/ee/api/lint.html#validate-a-projects-ci-configuration
```

Comment:
I think the section before this one is relevant as we cover both GET/POST, so from my perspective we could just link them all above in the module docstrings.

https://docs.gitlab.com/ee/api/lint.html#validate-a-ci-yaml-configuration-with-a-namespace

Reply:
I removed the anchor tags from both. Forgot to update them when I added support for `create` (`POST`)

Reply:
I do like to have them with this class as it is the one class that is not documented in https://docs.gitlab.com/ee/api/projects.html  So makes it easier for people to realize that when looking at the code.

Reply:
Ah I see. I just prefer docstrings (even if in classes) over comments in general, but no blocker :)

Reply:
Oh. I can make it a doc-string...